package.json

```diff
@@ -8,6 +8,7 @@
       "@material-ui/styles": "^4.9.6",
       "apollo-boost": "^0.4.7",
       "axios": "^0.19.2",
+      "git-mob": "^1.1.0",
```

Comment:
We should not have `git-mob` in our project. That's a dev tool intended to be used as a globally installed package.

Reply:
Yes, I'll fix that now

Reply:
Done

Reply:
Removed it again, checks out